pkg/kubelet/kubelet_node_status.go

```diff
@@ -172,6 +175,36 @@ func (kl *Kubelet) reconcileHugePageResource(initialNode, existingNode *v1.Node)
 	return requiresUpdate
 }
 
+// reconcileSwapResource will update swap capacity and remove swap size no longer supported
+func (kl *Kubelet) reconcileSwapResource(initialNode, existingNode *v1.Node) bool {
+	capacity, resourceIsCapacity := existingNode.Status.Capacity[v1.ResourceSwap]
+	if resourceIsCapacity && !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.NodeSwap) {
```

Comment:
```suggestion
	capacity, found := existingNode.Status.Capacity[v1.ResourceSwap]
	if found && !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.NodeSwap) {
```

I think is the golang convention, I didn't understand what `resourceIsCapacity` meant

Reply:
What behaviour do we want if this is not found? In that case, wouldn't we still want to delete this from the node capacity on update?